kubejs/server_scripts/base/recipes/industrialforegoing/laser_drill_ore.js

```diff
@@ -0,0 +1,780 @@
+ServerEvents.recipes((event) => {
+    const id_prefix = 'enigmatica:base/industrialforegoing/laser_drill_ore/';
+    const default_blacklist =  [
```

Comment:
Biome lists are already set up in constants/biomes

`biomes.in_the_nether` returns an array of all nether biomes, for example.

Reply:
Being completely honest, I didn't realize this blacklist is just end biomes for some reason >.>
Funnily enough I did use `biomes` constant in whitelist 😅
Gonna replace that with `biomes.in_the_end`